server/db/Seeder.js

```diff
@@ -15,17 +15,19 @@ class Seeder {
     let adoptablePets = contents.data.adoptablePets;
 
     try{
-      for (let i = 0; i < petTypes.length; i++) {
-        let query = 'INSERT INTO pet_types (type, img_url, description) VALUES ($1, $2, $3)';
-        let result = await pool.query(query, [petTypes[i].type, petTypes[i].imgURL, petTypes[i].description]);
+      for( let x = 0; x < petTypes.length; x++){
+          let petType = petTypes[x];
+          let query = 'INSERT INTO pet_types (type, img_url, description) VALUES ($1, $2, $3) RETURNING id';
+          let result = await pool.query(query, [petType.type, petType.imgURL, petType.description]);
+          console.log(result.rows[0]);
+        }
+
+      for( let x = 0; x < adoptablePets.length; x++){
+        let pet = adoptablePets[x];
+        let idArr = await pool.query('INSERT INTO adoptable_pets (name, img_url, age, vaccination_status, adoption_story, available_for_adoption, pet_type_id) VALUES ($1, $2, $3, $4, $5, $6, $7) RETURNING id', [pet.name, pet.img_url, pet.age, pet.vaccination_status, pet.adoption_story, pet.available_for_adoption, pet.pet_type_id]);
+        console.log(idArr.rows[0]);
```

Comment:
19-22 is indented with a size of 4 whereas 26-28 is indented with a size of 2.  Update the first for block to be consistent with a tab size of 2